recipes/base_install.rb

```diff
@@ -199,3 +199,8 @@
     end
   end
 end
+
+# Intel Runtime Libraries
+if (node['platform'] == 'centos' && node['platform_version'].to_i >= 7)
+  include_recipe "aws-parallelcluster::intel_install"
```

Comment:
I see that _efa_install, _lustre_install etc have an underscore prefix, intel_install not.

Reply:
I agree, thank you for the explanation.